docroot/sites/default/settings.php

```diff
@@ -807,3 +807,5 @@
   include_once './modules/contrib/fast404/fast404.inc';
   fast404_preboot($settings);
 }
+
+$config['acquia_connector.settings']['subscription_data']['active'] = FALSE;
```

Comment:
This setting can be removed. I think it overrides the settings from acquia.settings.php